modules/declarative/pages/policy.adoc

```diff
@@ -0,0 +1,293 @@
+include::{mod-loc}attributes.adoc[]
+
+//Category: skupper-policy
+// Type: assembly
+[id="skupper-policy"] 
+= Securing a {service-network}
+
+By default, {skupper-name} includes many security features.
+If you apply the {policy-system}, all communication is denied, and you can specify granular policies to allow your {service-network} operate.
+
+{skupper-name} provides default, built-in security that scales across clusters and clouds. 
+In a {service-network}, the communication between sites is performed by {skupper-router}s and is secured with mutual TLS using a private, dedicated certificate authority (CA). 
+Each {skupper-router} is uniquely identified by its own certificate.
+This means that the {service-network} is isolated from external access, preventing security risks such as lateral attacks, malware infestations, and data exfiltration.
+
+
+// Type: concept
+[id="about-skupper-policies"] 
+== About the {policy-system}
+
+After installing the {policy-system} Custom Resource Definition (CRD), you need to configure one or more of the following items using custom resources (CRs) to enable communication:
+
+Allow incoming links:: Use `AllowIncomingLinks` to enable users create tokens and configure incoming links.
+
+Allow outgoing links to specific hosts:: Use `allowedOutgoingLinksHostnames` to specify hosts that users can create links to.
+
+Allow services:: Use `allowedServices` to specify which services can be created or used on the {service-network}.
+
+Allow resources to be exposed:: Use `allowedExposedResources` to specify which service bindings can be created or used on the {service-network}.
+
+NOTE: Each setting for the {policy-system} can be applied to one or more namespaces.
+
+For example, the following policy CR disallows outgoing links for all namespaces, but all other {skupper-name} capabilities are allowed:
+
+[source,yaml]
+----
+apiVersion: skupper.io/v1alpha1
+kind: SkupperClusterPolicy
+metadata:
+  name: cluster-policy-sample-01
+spec:
+  namespaces:
+    - "*"
+  allowIncomingLinks: true
+  allowGateway: true
+  allowedExposedResources:
+    - "*"
+  allowedOutgoingLinksHostnames: []
+  allowedServices:
+    - "*"
+----
+
+`namespaces`:: One or more patterns to specify the namespaces that this policy applies to.
```

Comment:
It's worth mentioning that values for namespaces can also be [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).